README.md

```diff
@@ -1 +1,29 @@
-# examples
\ No newline at end of file
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/serverless-express/mainline/vendia-logo.png" alt="vendia logo" width="100px">
```

Comment:
Probably add the logo to this repo and link to that

Reply:
👍 just added